auth0_flutter_platform_interface/lib/src/credentials-manager/credentials_manager_platform.dart

```diff
@@ -36,6 +35,11 @@ abstract class CredentialsManagerPlatform extends PlatformInterface {
     throw UnimplementedError('getCredentials() has not been implemented');
   }
 
+  /// Retrieves the credentials from the native storage.
+  Future<UserInfo> getIDTokenContents(final CredentialsManagerRequest request) {
```

Comment:
The `Credentials` class [uses](https://github.com/auth0/auth0-flutter/blob/main/auth0_flutter_platform_interface/lib/src/credentials.dart#L48) a `user` property for this. Since this method here is exposing exactly the same functionality (parsing and returning the info contained in the ID token), we should stick to the same naming convention here, otherwise it could be confusing (different names for the same thing may suggest these are actually different things, or that there is a substantive difference).